kaldb/src/main/java/com/slack/kaldb/histogram/FixedIntervalHistogramImpl.java

```diff
@@ -95,24 +98,6 @@ private Optional<HistogramBucket> findMatchingBucket(HistogramBucket matchingBuc
     return Optional.empty();
   }
 
-  /**
-   * Runs a binary search to find the correct bucket for the value being inserted and return it's
```

Comment:
Can we add a unit test comparing the output from this method and the proposed method. That would be a cheap way to make sure that the outputs are identical. 

Also, can we ensure that this method would work in the aggregation path also?

Reply:
As discussed via Slack, we verified the new version is generating identical outputs with an uncommitted test prior to removing this old version.